common/components/BalanceSidebar/EquivalentValues.tsx

```diff
@@ -1,177 +1,212 @@
-import * as React from 'react';
-import BN from 'bn.js';
+import React from 'react';
 import translate from 'translations';
-import { State } from 'reducers/rates';
-import { rateSymbols, TFetchCCRates } from 'actions/rates';
+import { UnitDisplay, Spinner } from 'components/ui';
+import Select from 'react-select';
+import { TFetchCCRates, rateSymbols } from 'actions/rates';
+import { chain, flatMap } from 'lodash';
+import { NetworkConfig } from 'config';
+import { State as RatesState } from 'reducers/rates';
 import { TokenBalance } from 'selectors/wallet';
 import { Balance } from 'libs/wallet';
-import { NetworkConfig } from 'config';
-import { ETH_DECIMAL, convertTokenBase } from 'libs/units';
-import Spinner from 'components/ui/Spinner';
-import UnitDisplay from 'components/ui/UnitDisplay';
 import './EquivalentValues.scss';
 
-const ALL_OPTION = 'All';
+const isFiat = (rate: string): boolean => {
+  return rateSymbols.slice(0, 4).includes(rate as any);
```

Comment:
This is quite fragile -- I would rather have `rateSymbols` become an object with an `isFiat` boolean.